website/docs/authenticating_via_msi.html.markdown

```diff
@@ -7,17 +7,17 @@ description: |-
 
 ---
 
-# Authenticating to Azure Resource Manager using Managed Service Identity
+# Azure Provider: Authenticating using Managed Service Identity
 
-Terraform supports authenticating to Azure through Managed Service Identity, Service Principal or the Azure CLI.
+Terraform supports authenticating to Azure through Managed Service Identity, a Service Principal, or the Azure CLI.
 
-Managed Service Identity can be used to access other Azure Services from within a Virtual Machine in Azure instead of specifying a Service Principal or Azure CLI credentials.
+Instead of specifying a Service Principal or Azure CLI credentials, Managed Service Identity can be used to access other Azure Services from within an Azure VM. 
```

Comment:
what do you think about making this:

```suggestion
Instead of specifying a Service Principal or Azure CLI credentials, Managed Service Identity can be used to interact with other Azure Services from within a Virtual Machine in Azure. 
```

Reply:
Much better. So corrected.